docs/API/index.md

```diff
@@ -1,10 +1,10 @@
 ---
 id: "index"
-title: "mobx-state-tree"
+title: "mobx-state-tree - v3.16.0"
```

Comment:
probably related to deps changes from:

https://github.com/mobxjs/mobx-state-tree/pull/1539/files#diff-8ee2343978836a779dc9f8d6b794c3b2L9116-L9148

Reply:
I've tried to set 
https://typedoc.org/guides/options/#includeversion
to false, but it didn't help.
I think it's hard coded somehow with our theme